### DISCUSSION TITLE
Tawk.to Chat Bot and Affiliation

### DISCUSSION THREAD

**Discussion by ATLCOTech** [code4recovery/12-step-meeting-list#1410] [General]
Hi! Does anyone have any experience strength and hope on feedback from the groups on using chat bots? We have 1 group who feels there is an issue of affiliation (see screen shot at the bottom). 
![image](https://github.com/code4recovery/12-step-meeting-list/assets/147879274/126a9af2-3b94-465c-84f9-27862d5df849)

**Comment by joshreisner** [comment]
hi @ATLCOTech it might be good to get a little more info about the specific objection, perhaps they don't like the way the chat feature is branded and has the message at the bottom? you might ask if they like the presentation of the chat feature on https://aasfmarin.org/ or https://aasanjose.org better? the use a different service which is a little more discreet
